test.json

```diff
@@ -2,7 +2,7 @@
   "source": "./package.json",
   "scripts": {
     "run": "esy '@test' x OniUnitTestRunner",
-    "run-ci": "esy '@test' x OniUnitTestRunnerCI",
+    "run-ci": "#{os == 'linux' ? 'xvfb-run' : ''} esy '@test' x OniUnitTestRunnerCI",
```

Comment:
Thanks for figuring this out!

Windows CI is failing with this:
```
========================== Starting Command Output ===========================
"C:\windows\system32\cmd.exe" /D /E:ON /V:OFF /S /C "CALL "D:\a\_temp\5df81a8d-b1b5-4fb0-b381-158024b29307.cmd""
error: Exec format error
  
esy: exiting due to errors above
```
https://dev.azure.com/onivim/oni2/_build/results?buildId=14382&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=d6c700ea-12df-53fd-bd57-c3cacedc9ab9&l=11


I wonder if it is due to the leading whitespace here?

Reply:
Ah that would make sense -- I can fix that!

Reply:
Maybe we can try: 
```
"run-ci": "#{os == 'linux' ? 'xvfb-run esy' : 'esy'} '@test' x OniUnitTestRunnerCI",
```